src/app/view/components/_common/dot-site-selector/dot-site-selector.component.ts

```diff
@@ -57,6 +58,10 @@ export class DotSiteSelectorComponent implements OnInit, OnChanges, OnDestroy {
         private dotEventsService: DotEventsService
     ) {}
 
+    get currentSite(): Observable<Site> {
```

Comment:
```suggestion
    get currentSite$(): Observable<Site> {
```

I forgot about this but is a good practice to append a `$` to the properties that are `Observables`